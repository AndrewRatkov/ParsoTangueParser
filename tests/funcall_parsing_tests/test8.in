h(30*30==900,f())