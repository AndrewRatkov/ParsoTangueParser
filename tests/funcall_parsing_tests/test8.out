[CALL VOID h]------->{incorrect argument 1: expected StringExpr, got IntExpr}
